src/libs/ReportUtils.js

```diff
@@ -1370,7 +1370,8 @@ function buildOptimisticExpenseReport(chatReportID, policyID, payeeEmail, payeeA
  * @returns {Array}
  */
 function getIOUReportActionMessage(type, total, comment, currency, paymentType = '', isSettlingUp = false) {
-    const amount = NumberFormatUtils.format(preferredLocale, total / 100, {style: 'currency', currency});
+    const currencyUnit = CurrencyUtils.getCurrencyUnit(currency);
```

Comment:
I think this is incorrect, the `total` is always in "cents" independent of the currency, so it should have remained divided by 100. Correcting here: https://github.com/Expensify/App/pull/24175